tendrl/notifier/objects/definition/notifier.yaml

```diff
@@ -48,6 +48,9 @@ namespace.alerting:
         source:
           help: 'The process raising the alert'
           type: String
+        delivery:
```

Comment:
I think we discussed that you would be enhancing this to handle plugin and end-point specific manner..
Have you raised an issue to track it so that we don't miss it out..